.github/workflows/artifacts.yml

```diff
@@ -30,8 +30,7 @@ jobs:
       uses: goreleaser/goreleaser-action@v2
       with:
         install-only: true
-        # Newer goreleaser version doesn't allow builds with "latest" tag
-        version: v0.179.0
+        version: latest
```

Comment:
I am bit confused on this change. Everywhere else we are using `v9.99.9-dev` version where as here we are using latest version.

Also, we have removed the earlier comments.
```
# Newer goreleaser version doesn't allow builds with "latest" tag
```
So Goreleaser accepts latest version now ?

Reply:
@sanketsudake the version we have mentioned here is the Goreleaser version we use in CI - not the version we build images with.
Goreleaser doesn't support image builds with `latest` tag, that's why we have replaced tag `latest` with `v9.99.9-dev` tags for BotKube build images.